mono/mini/Makefile.am.in

```diff
@@ -724,26 +747,40 @@ gsharedvtcheck:
 
 fullaot_regtests = $(regtests) aot-tests.exe $(if $(GSHAREDVT),gshared.exe)
 
-FULLAOT_LIBS = \
-	mscorlib.dll \
-	System.Core.dll \
-	System.dll \
+FULLAOT_LIBS_MOBILE_STATIC_BLACKLIST = \
```

Comment:
These are libs that we don't compile in the bcl on the mobile_static profile. They're not compiled on mobile platforms.